internal/core/api.rs

```diff
@@ -299,7 +299,7 @@ impl<F: FnMut(RenderingState, &GraphicsAPI)> RenderingNotifier for F {
 }
 
 /// This enum describes the different error scenarios that may occur when the application
-/// registers a rendering notifier on a [`crate::Window`](struct.Window.html).
+/// registers a rendering notifier on a `slint::Window`.
```

Comment:
it's sad too lose this link.

Reply:
Ah, then let me restore it.I don't mind the visible brackets in C++ :)

Reply:
Oh wait, I must've looked at a cached page - it's indeed not possible because of the missing crate. :( Goodbye link.